README.md

```diff
@@ -96,6 +92,11 @@ import { Button } from 'flotiq-components-react';
 <Button label="Hello Component!"/>
 ```
 
+## NPM publish
+
+To publish a new package in NPM, you need to update the version in the packages.json and packages-lock.json files and then commit the changes with the message "Release x.y.z".
```

Comment:
there is no package-lock.json, only yarn.lock and it doesn't have a version inside
```suggestion
To publish a new package in NPM, you need to update the version in the packages.json file and then commit the changes with the message "Release x.y.z".
```